uat/behat.yml

```diff
@@ -1,57 +1,15 @@
+imports:
+  - ../includes/extensions/drupal.extension.yml
+  - ../includes/extensions/mink.extension.yml
+  - behat.local.yml
+
+## Default Profile Settings (Sites)
 default:
   paths:
     features: 'features'
-#  suites:
   extensions:
     Behat\MinkExtension\Extension:
-      goutte: ~
-      selenium2: ~
-      base_url: https://sites-uat.stanford.edu/drupal7-update-status/
-      files_path: "."
+      base_url: <<< THE FULL URL TO THE SITE HERE eg: https://sites.stanford.edu/jumpstart >>>
```

Comment:
This is the default profile and should be set to: https://sites-uat.stanford.edu/drupal7-update-status/